rfps/hedera-and-filecoin.md

```diff
@@ -1,3 +1,8 @@
+NOTE (09/20/2021): The following RFP has now been closed. 
+If you wish to apply for a Grant to fund your work, please apply instead under the [Filecoin Grant Program](https://github.com/filecoin-project/devgrants). There are a variety of grants to fulfil the development stage you are at. 
```

Comment:
```suggestion
If you wish to apply for a grant to fund your work, please apply instead via the [Filecoin Dev Grants Program](https://github.com/filecoin-project/devgrants). 
```

1. `grant` should be lowercase.
2. the markdown format doesn't respect line breaks (unless there's a blank line in between), so lines 1 and 2 will run together. i think that's ok.
3. updated name to `Filecoin Dev Grants` program